dev/benchmarking/make_machine_json.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -e
+OUTFILE=machine.json
+
+echo "Making ${OUTFILE}"
+echo "**NOTE: This command fails on virtual machines and Windows.**"
+echo "Please type GPU details here (or manually modify ${OUTFILE} later)."
+read -p "GPU model name (or <enter>): " gpu_model_name
+read -p "GPU core count (or <enter>): " gpu_core_count
+read -p "GPU RAM (in bytes, or <enter>): " gpu_ram_bytes
```

Comment:
Thank you for helpful feedback. I changed it to megabytes to keep it an integer. I like your idea of using the nvidia command rather than manual interaction for the chip details.

Reply:
Actually modified the table per pearu's comments below to have only these columns: 'gpu_part_number', 'gpu_model_name', 'gpu_nvidia_smi'